components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/resources/admin-api.yaml

```diff
@@ -1809,494 +1448,103 @@ paths:
-          users **only at a migration process**.
-        schema:
-          type: string
-      responses:
-        200:
-          description: |
-            OK.
-            Application list returned.
-          headers:
-            Content-Type:
-              description: |
-                The content type of the body.
-              schema:
-                type: string
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/ApplicationList'
-        304:
-          description: |
-            Not Modified.
-            Empty body because the client has already the latest version of the requested resource (Will be supported in future).
-          content: {}
-        400:
-          $ref: '#/components/responses/BadRequest'
-        406:
-          $ref: '#/components/responses/NotAcceptable'
-      security:
-      - OAuth2Security:
-        - apim:admin
-        - apim:app_owner_change
-        - apim:app_import_export
-      x-code-samples:
-      - lang: Shell
-        source: 'curl -k -H "Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8"
-          "https://127.0.0.1:9443/api/am/admin/v2/applications"'
-
-  ######################################################
-  # The "Individual Application" resource APIs
-  ######################################################
-  /applications/{applicationId}:
-    delete:
-      tags:
-      - Applications
-      summary: |
-        Delete an Application
-      description: |
-        This operation can be used to delete an application by specifying its id.
-      parameters:
-      - $ref: '#/components/parameters/applicationId'
-      - $ref: '#/components/parameters/If-Match'
-      responses:
-        200:
-          description: |
-            OK.
-            Resource successfully deleted.
-          content: {}
-        202:
-          description: |
-            Accepted.
-            The request has been accepted.
-          headers:
-            Location:
-              description: |
-                Location of the existing Application.
-              schema:
-                type: string
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/WorkflowResponse'
-        404:
-          $ref: '#/components/responses/NotFound'
-        412:
-          $ref: '#/components/responses/PreconditionFailed'
-      security:
-      - OAuth2Security:
-        - apim:admin
-        - apim:app_import_export
-      x-code-samples:
-      - lang: Shell
-        source: 'curl -k -X DELETE -H "Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8"
-          "https://127.0.0.1:9443/api/am/admin/v2/applications/0a043c2b-ee75-4ef3-9e1c-fc2610ccfa8b"'
-
-  /applications/{applicationId}/change-owner:
-    post:
-      tags:
-      - Application
-      summary: Change Application Owner
-      description: |
```

Comment:
Was this removal intentional?

Reply:
This is not a remove actually. Git diff that is displayed is not correct. These code segments are unchanged.

Reply:
https://github.com/wso2/carbon-apimgt/pull/9986/files#diff-833d66b2c304471172f7cf9a088df847a742e0f4832cb0ac10e3818165eae730R1616